examples/provider/config.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kops.crossplane.io/v1alpha1
+kind: ProviderConfig
+metadata:
+  name: example
+spec: {}
```

Comment:
are there further details that could be added into this ProviderConfig example that would help people know how to use it?

Reply:
Currently, the providerConfig should remain empty. i wanted to keep it for adding capabilities into it in the future